src/fuzz_tests.zig

```diff
@@ -63,38 +67,42 @@ pub fn main() !void {
             assert(cli_args.events_max == null);
             try smoke();
         },
-        inline else => |fuzzer| try @field(Fuzzers, @tagName(fuzzer)).main(.{
-            .seed = seed,
-            .events_max = cli_args.events_max,
-        }),
+        inline else => |fuzzer| {
+            var timer = try std.time.Timer.start();
+            try @field(Fuzzers, @tagName(fuzzer)).main(.{
+                .seed = seed,
+                .events_max = cli_args.events_max,
+            });
+            log.info("done in {}", .{std.fmt.fmtDuration(timer.lap())});
+        },
     }
 }
 
 fn smoke() !void {
     var timer_all = try std.time.Timer.start();
     inline for (comptime std.enums.values(FuzzersEnum)) |fuzzer| {
-        const fuzz_args = switch (fuzzer) {
+        const events_max = switch (fuzzer) {
             .smoke => continue,
             // TODO: At one point, these were too slow to run, but surely there is _some_ way
             // to run them fast enough?
-            .lsm_forest,
             .lsm_cache_map,
             .lsm_manifest_log,
             .vsr_free_set,
             => continue,
 
-            .lsm_tree => .{ .seed = 123, .events_max = 400 },
-            .vsr_superblock => .{ .seed = 123, .events_max = 3 },
+            .lsm_forest => 10000,
+            .lsm_tree => 400,
+            .vsr_superblock => 3,
 
             inline .ewah,
             .lsm_segmented_array,
             .vsr_journal_format,
             .vsr_superblock_quorums,
-            => .{ .seed = 123, .events_max = null },
+            => null,
         };
 
         var timer_single = try std.time.Timer.start();
-        try @field(Fuzzers, @tagName(fuzzer)).main(fuzz_args);
+        try @field(Fuzzers, @tagName(fuzzer)).main(.{ .seed = 123, .events_max = events_max });
```

Comment:
Since we override the seed here, maybe best to move the printing of "Fuzz seed:" into the non-smoke case?

Reply:
Excellent catch, done!

This reminds me that we should use that trick with "use git hash as a seed" here, will do that further up the stack of PRs!